docs/basic_configuration.rst

```diff
@@ -0,0 +1,66 @@
+There are much more options, see [[configuration]].
+
+Preparations
+============
+Get a supported map package
+---------------------------
+Navit can use different :doc:`maps` formats, including free :ref:`OpenStreetMap` data. In order to use one of these maps, download a map of your desired area and store it into a local folder (such as navit/maps or /usr/share/navit/maps).
+
+Install TTS
+-----------
+To get speech support, you need to install a text-to-speech tool such as **espeak**, **mbrola** or **festival**. These tools can be invoked from the command line. Test your setup by invoking the tools manually e.g.
+
+.. code-block:: bash
+
+  espeak "This is a text!"
+
+On some systems Navit comes with integrated espeak support, so you don't need to download it separately.
+
+Connect GPS
+-----------
+Now please connect your GPS. The exact procedure for this varies depending on the type of GPS device you are using and how you connect it to your computer. On Linux / Unix systems, your GPS should typically show up as a character device, i.e., you will find an entry in the /dev folder corresponding to your GPS device. Again, the file name depends on the type and connection method of your GPS receiver. See [[Connecting a GPS receiver]] for details.
```

Comment:
You probably don't need to say `please` in `Now please`, it looks a bit too much.
The idea of the doc is to give instructions so I think we'd be good without it.

Reply:
Indeed. This was copied from the wiki.